src/Broadway/EventStore/DBALEventStore.php

```diff
@@ -170,4 +194,30 @@ private function deserializeEvent($row)
             DateTime::fromString($row['recorded_on'])
         );
     }
+
+    private function extractId($id)
+    {
+        if ($this->useBinary) {
+            try {
+                return Uuid::fromString($id)->getBytes();
+            } catch (\Exception $e) {
```

Comment:
Not handling the exception, but swallowing it leads to undefined behavior that might be hard to debug. I'm wondering if you should rethrow a proper exception.